plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/TickCommand.java

```diff
@@ -0,0 +1,121 @@
+
+    // <--[command]
+    // @Name tick
+    // @Syntax tick [rate/step/sprint/freeze/reset] (amount:<amount>/cancel)
+    // @Required 1
+    // @Maximum 2
+    // @Short Controls the server's tick rate.
+    // @Group world
+    //
+    // @Description
+    // Controls the server's tick rate. MC versions 1.20+ only.
+    //
+    // The tick rate affects things such as entity movement (including player movement), entity animations, plant growth, etc.
+    // Be careful when setting the tick rate, as a high tick rate can overload the server.
+    // If tick rate is high, crops will grow faster, animations will speed up, and so forth.
+    // Conversely, if the tick rate is low, crops will grow slower, animations will slow down, and entity movement (including player movement) will slow down and appear to be in slow motion.
+    // For a list of all the things that the tick rate affects, see <@link url https://minecraft.wiki/w/Tick>
+    //
+    // To change the tick rate, use the 'rate' argument and input the amount using 'amount'. The tick rate must be a number between 1.0 and 10000.0 (inclusive).
+    // To reset the tick rate to the normal value (20.0), use the 'reset' argument.
+    //
+    // To freeze the tick rate, use the 'freeze' argument. To unfreeze, add the 'cancel' argument.
+    // Freezing the tick rate will cause all entities (except for players and entities that a player is riding) to stop ticking.
+    // This means that entity movement and animations will stop, as well as stop things like crop growth, daylight cycle, etc.
+    //
+    // To step the tick rate while the server is frozen for a specific amount of time, use the 'step' argument and input the amount of ticks using 'amount'.
+    // Tick rate stepping is allowing the tick rate to resume for a specific amount of ticks while the server is frozen.
+    // After the amount of specified ticks have passed, the tick rate will refreeze.
+    // If the server is not frozen when trying to use the 'step' argument, nothing will happen.
+    //
+    // To make the tick rate as fast as possible for a specific amount of time, use the 'sprint' argument and input the amount of ticks using 'amount'.
+    // The tick rate will increase as much as possible without overloading the server for the amount of specified ticks.
+    // For example, if you want to sprint 200 ticks, the tick rate will run 200 ticks as fast as possible and then return to the previous tick rate.
+    // To stop stepping or sprinting early, use the 'cancel' argument.
+    //
+    // The tick rate resets to 20.0 on server restart.
+    // For information about tick rate arguments, see <@link url https://minecraft.wiki/w/Commands/tick>
+    //
+    // @Warning Be careful, this command will affect plugins that depend on tick rate and may cause them to break.
+    // @Warning For example, setting the tick rate to 1 will cause the <@link event tick> event to fire once per second.

```

Comment:
This will be 2 separate values (or only one of them, not sure what's the parsing like internally) - need to have it all under one key like `@Description`